docs/EN_US/HPCCSystemAdmin/SA-Mods/WUTool.xml

```diff
@@ -99,6 +113,287 @@
+        </row>
+
+        <row>
+          <entry>depend &lt;wu&gt; ?&lt;activity&gt;:startTime</entry>
+
+          <entry>Which dependencies take a large % of the start time for this
+          activity</entry>
+        </row>
+
+        <row>
+          <entry>help &lt;command&gt;</entry>
+
+          <entry>More help on a command</entry>
+        </row>
+      </tbody>
+    </tgroup>
+  </informaltable>
+
+  <para>If the<emphasis role="bold"> [CASSANDRASERVER=ip]</emphasis> option is
+  specified, you can specify some connection options including:</para>
+
+  <para><informaltable colsep="1" frame="all" rowsep="1">
+      <tgroup cols="2">
+        <colspec align="left" colwidth="175.55pt" />
+
+        <colspec />
+
+        <tbody>
+          <row>
+            <entry>CASSANDRA_KEYSPACE</entry>
+
+            <entry>The default is hpcc</entry>
+          </row>
+
+          <row>
+            <entry>CASSANDRA_USER</entry>
+
+            <entry>Username</entry>
+          </row>
+
+          <row>
+            <entry>CASSANDRA_PASSWORD</entry>
+
+            <entry>Password</entry>
+          </row>
+
+          <row>
+            <entry>CASSANDRA_PARTITIONS</entry>
+
+            <entry>Use if creating a new repository</entry>
+          </row>
+
+          <row>
+            <entry>CASSANDRA_PREFIXSIZE</entry>
+
+            <entry>Use if creating a new repository</entry>
+          </row>
+
+          <row>
+            <entry>TRACELEVEL</entry>
+
+            <entry></entry>
+          </row>
+
+          <row>
+            <entry>archive &lt;workunits&gt;</entry>
+
+            <entry>Archive specified Workunits to xml files. The following
+            options are supported: <para>[TO=&lt;directory&gt;]
+            </para><para>[DEL=1] </para><para>[DELETERESULTS=1]
+            </para><para>[INCLUDEFILES=1]</para></entry>
+          </row>
+        </tbody>
+      </tgroup>
+    </informaltable></para>
+
+  <para>The following table provides further information for the wutool
+  utility issued with the action=info parameter:</para>
+
+  <informaltable colsep="1" frame="all" rowsep="1">
+    <tgroup cols="2">
+      <colspec align="left" colwidth="175.55pt" />
+
+      <colspec />
+
+      <tbody>
+        <row>
+          <entry colsep="0"><emphasis role="bold">info
+          parameters</emphasis></entry>
+
+          <entry></entry>
+        </row>
+
+        <row>
+          <entry>info &lt;workunits&gt; &lt;filter&gt;</entry>
+
+          <entry>This command provides filtered access to statistics and other
+          information from a workunit. <para>The filter can include the
+          following elements (those denoted by * can be repeated):
+          </para></entry>
+        </row>
+
+        <row>
+          <entry colsep="0"></entry>
+
+          <entry><emphasis role="bold">Which scopes are
+          matched</emphasis>:</entry>
+        </row>
+
+        <row>
+          <entry><emphasis role="white">.
```

Comment:
Is this a kludge to force a space by having a dot in white? How would this work in the future when we support dark mode in HTML? 
An alternate way would be to use **role="wsig"** which forces the text to monospaced font and allows indenting by spaces.